docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -10,7 +10,9 @@ Teleport.
 
 ## Prerequisites
 
-- A GitHub organization with at least one team.
+- A GitHub organization with at least one team. <ScopedBlock scope="oss">This organization must not have external SSO setup, or Teleport
+will refuse to create the GitHub authentication connector. If you want to use a GitHub organization that uses external SSO, consider purchasing
+a Teleport Enterprise license.</ScopedBlock>
```

Comment:
@xinding33 Can you review this wording please? This will appear in our Github SSO docs.

Reply:
@capnspacehook @jakule Let's remove the "If you want to use a GitHub organization that uses external SSO, consider purchasing a Teleport Enterprise license." sentence and then :shipit:

Reply:
Done